community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/RecordProcessorStep.java

```diff
@@ -31,13 +36,15 @@
 {
     private final RecordProcessor<T> processor;
     private final boolean endOfLine;
+    private final Collection<StatsProvider> additionalStatsProviders;
 
     public RecordProcessorStep( StageControl control, String name, int workAheadSize, int movingAverageSize,
-            RecordProcessor<T> processor, boolean endOfLine )
+            RecordProcessor<T> processor, boolean endOfLine, StatsProvider... additionalStatsProviders )
     {
         super( control, name, workAheadSize, movingAverageSize, 1 );
         this.processor = processor;
         this.endOfLine = endOfLine;
+        this.additionalStatsProviders = asList( additionalStatsProviders );
```

Comment:
Forgot this place when changing the previously temporary implementation. I'd guess I was a little too quick yesterday evening